e2e/cypress/tests/support/ui/account_settings_modal.d.ts

```diff
@@ -27,7 +27,7 @@ declare namespace Cypress {
          *       // Do something here
          *   });
          */
-        uiOpenProfileModal(): Chainable;
+        uiOpenProfileModal(section?: string): Chainable<void>;
```

Comment:
```suggestion
        uiOpenProfileModal(section?: string): Chainable<JQuery<HTMLElement>>;
```

uiOpenProfileModal is returning HTML Element.